public/index.js

```diff
@@ -27,7 +27,8 @@ async function initMap()
             newButton.style.height = 90;
             newButton.style.backgroundColor = '#' + route.routeColor
             newButton.style.color = 'white';
-            newButton.addEventListener('click', async () => await DisplayRoute(route.routeId, '#' + route.routeColor));
+            newButton.classList.add(route.routeShortName);

```

Comment:
Can we make this so it uses the ID property of the button to look it up instead of a class?

Reply:
Also, can we make the ID based on the route.routeId property? Something like 'button-route-<id>'? That way we don't need to pass the short name into the event listener function